wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss

```diff
@@ -5,7 +5,7 @@
 */
 
 body:not(.trac):not(.home-page) #main {
-	margin-top: 0.5rem;
+	margin-top: 2rem;
```

Comment:
This is a more reliable way of controlling the space at the top of every page, as there isn't always a `section`, eg. on regular pages like Online Workshops